src/guide/advanced/triggers.md

```diff
@@ -39,29 +39,15 @@ A trigger has roughly the following form:
 struct Trigger {
   id: TriggerId,
   action: Action,
-  metadata: Metadata,
 }
 ```
 
 ### `Trigger.id`
 
 The `TriggerId` is a simple wrapper around a single `Name`, i.e. a string
-with no whitespaces and no reserved characters (`@`, `#`).
-
-::: info
-
-<!-- Check: a reference about future releases or work in progress -->
-
-In the future, we shall add scoped triggers, and the id will be expanded to
-be either a global trigger, or a trigger with a domain name. This is what
-determines the scope of the trigger.
-
-:::
-
-### `Trigger.metadata`
-
-This `Metadata` is the same kind of `Metadata` that can be attached to
-accounts, domains, assets, or transactions.
+with no whitespaces and no reserved characters (`@`, `#`, `$`).
+[Domain-scoped triggers](#domain-scoped-triggers) use `$` to separate the
+domain id and the trigger id.
```

Comment:
This just tells us nothing. 
```suggestion
A typical domain-scoped trigger looks like `trigger_id$domain_name`, while a bare trigger looks like `@@trigger_id`, which makes these names easy to parse. 
```

Reply:
@appetrosyan , this style of trigger id is used only for displaying, not for parsing. I checked our code on `iroha2-dev`.
So should we fix it in out code? If so, then the fix should be backported as well

Reply:
```
The `TriggerId` is a simple wrapper around a single `Name`, a string with
no whitespaces and no reserved characters (`@`, `#`, `$`).

A typical [domain-scoped trigger](#domain-scoped-triggers) looks like
`trigger_id$domain_name`, while a bare trigger looks like `@@trigger_id`,
which makes these names easy to parse.
```